src/libraries/System.Resources.Writer/ref/System.Resources.Writer.cs

```diff
@@ -21,7 +21,7 @@ public sealed partial class ResourceWriter : System.IDisposable, System.Resource
         public System.Func<System.Type, string>? TypeNameConverter { get { throw null; } set { } }
         public void AddResource(string name, byte[]? value) { }
         public void AddResource(string name, System.IO.Stream? value) { }
-        public void AddResource(string name, System.IO.Stream? value, bool closeAfterWrite) { }
+        public void AddResource(string name, System.IO.Stream? value, bool closeAfterWrite = false) { }
```

Comment:
this is a breaking change right? This seems like it will conflict with, 

```cs
 public void AddResource(string name, System.IO.Stream? value) { }
```

Right?

Reply:
I added this here: https://github.com/dotnet/corefx/pull/36906

It's not technically breaking.  Same source binds to same method without `closeAfterWrite` parameter.  Compiler prefers methods with fewer params before considering defaults.  The default will never be used in this case (but it is in another case that shares the same source file but doesn't have the method without string).

Reply:
I see, ok sounds good.